python/flyft/_flyft/mesh.cc

```diff
@@ -50,5 +55,7 @@ void bindMesh(py::module_& m)
         .def("bin", &Mesh::bin)
         .def("lower_bound",&Mesh::lower_bound)
         .def("upper_bound",&Mesh::upper_bound)
+        .def("total_volume",py::overload_cast<>(&Mesh::volume, py::const_))
```

Comment:
We could decide to make this one a property and read only, it would make sense. I think this would work:
```suggestion
        .def_property_readonly("volume", py::overload_cast<>(&Mesh::volume, py::const_))
```